src/lib/core/sequence.c

```diff
@@ -1942,19 +1961,50 @@ bt_sequence_class_init (BtSequenceClass * const klass)
           "Set song object, the sequence belongs to", BT_TYPE_SONG,
           G_PARAM_CONSTRUCT_ONLY | G_PARAM_READWRITE | G_PARAM_STATIC_STRINGS));
 
+  /**
+   * A note on "length" and "len-patterns".
```

Comment:
If we ever want to refactor this, we could talk about "play-length" and "data-length". It is going to be a bit of a search and replace orgy though.